velox/docs/develop/operators.rst

```diff
@@ -600,7 +601,7 @@ results available before seeing all input.
   * - partitionKeys
     - Partition by columns.
   * - rowNumberColumnName
-    - Output column name for the row numbers.
+    - Optional output column name for the row numbers. If specified, the generated row numbers are returned in the output variables.
```

Comment:
> output variables

The term "variables" is not used anywhere else. Would it be possible to use 'columns' instead?